public-ui/src/app/views/ehr-view/ehr-view.component.html

```diff
@@ -127,19 +127,21 @@ <h5 class="col-xs-12 box-title">
           </div>
         </div>
         <div class="row">
-          <div class="col-md-4 grid-header flex-xs-middle"> {{dbDomain.domainDisplay}}</div>
-          <div class="col-md-4 grid-header flex-xs-middle"> Number of particpants </div>
+          <div class="col-md-2 grid-header flex-xs-middle"> {{dbDomain.domainDisplay}}</div>
+          <div class="col-md-4 grid-header flex-xs-middle"> Synonyms </div>
+          <div class="col-md-2 grid-header flex-xs-middle"> Particpants of <b> {{totalParticipants | number}}</b> </div>
           <div class="col-md-3 grid-header flex-xs-middle info-text"> <clr-icon class="" shape="info-standard" class="is-solid"></clr-icon> What is SNOMED? </div>
           <div class="col-md-1 grid-header flex-xs-middle info-text"> View graphs </div>
 
         </div>
         <div class="row expandable-grid-row"
              *ngFor="let r of items.concat(standardConcepts), let i = index"
              [ngClass]="{'expanded-grid-row': r.expanded}" >
-          <div class="col-md-4 flex-xs-middle" innerHtml="{{i+1}}. {{r.conceptName | highlightSearch : prevSearchText}}"> </div>
-          <div class="col-md-4 flex-xs-middle">
-            {{r.countValue | number }} out of {{totalParticipants | number}} participants
-
+          <div class="col-md-2 flex-xs-middle" innerHtml="{{i+1}}. {{r.conceptName | highlightSearch : prevSearchText}}
```

Comment:
Do you really need innerHtml? It should generally be avoided on account of risk of injection.

Reply:
Yes. It's been a while since i wrote this. For any pipe to change the markup and add the <mark/> tag as this does  it needs innerhtml.  Angular sanitizes the return from pipes and just displays the markup as text in the variable.   I never found a way around it.